frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx

```diff
@@ -330,6 +338,121 @@ function LinkedFlagSelector(): JSX.Element | null {
     )
 }
 
+function UrlTriggerForm(): JSX.Element {
+    const { cancelProposingUrlTrigger } = useActions(sessionReplayIngestionControlLogic)
+    const { isProposedUrlTriggerSubmitting } = useValues(sessionReplayIngestionControlLogic)
+
+    return (
+        <Form
+            logic={sessionReplayIngestionControlLogic}
+            formKey="proposedUrlTrigger"
+            enableFormOnSubmit
+            className="w-full flex flex-col border rounded items-center p-2 pl-4 bg-bg-light gap-2"
+        >
+            <div className="flex flex-row gap-2 w-full">
+                <LemonField name="matching">
+                    <LemonSelect options={[{ label: 'Regex', value: 'regex' }]} />
```

Comment:
do we need a select if the only option is regex?

Reply:
Not really. Reason for adding it is I wanted the api response and data to work the same way when we add other matching types, and the easiest way to do that here was to add the select with the single option.